File: Healthcare-app/android/MDBRealmDoctor/app/src/main/java/com/wekanmdb/storeinventory/model/code/Code.kt
package com.wekanmdb.storeinventory.model.code

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey
import org.bson.types.ObjectId;

open class Code(
    @PrimaryKey var _id: ObjectId = ObjectId(),

    var category: String? = null,

    var code: String? = null,

    var comments: String? = null,

    var name: String? = null,

    var active: Boolean? = true,

    var system: String? = null
): RealmObject() {}

File: Healthcare-app/ios/MDBRealmPatient/MDBRealmPatient/Code.swift
//
//  Condition.swift
//  MDBRealmPatient
//
//  Created by MaDHiVaNan on 5/2/22.
//

import Foundation
import RealmSwift

class Code: Object {
    @Persisted(primaryKey: true) var _id: ObjectId
    @Persisted var active: Bool?
    @Persisted var category: String?
    @Persisted var code: String?
    @Persisted var comments: String?
    @Persisted var name: String?
    @Persisted var system: String?
}
